plotter/heat.go

```diff
@@ -98,6 +98,11 @@ func (h *HeatMap) Plot(c draw.Canvas, plt *plot.Plot) {
 	}
 	// ps scales the palette uniformly across the data range.
 	ps := float64(len(pal)-1) / (h.Max - h.Min)
+	// for some degenerate data sets with h.Max == h.Min, this blows
+	// up and needs to be caught
+	if math.IsNaN(ps) {
```

Comment:
A flat heatmap itself isn't meaningful by itself, but as one of a set of heatmaps that show differences in data it shows the case of perfectly flat data and is a valid heatmap to compare to others.  Additionally, updating a heatmap continuously based on data flowing through can see cases where all data values are identically equal which is still a valid and well-formed case so i wouldn't expect this to cause a panic